pkg/asset/templates/kubelet-master.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kubelet
+  namespace: kube-system
+  labels:
+    k8s-app: kubelet
+    version: v1.2.4_inotify.3
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: kubelet
+        version: v1.2.4_inotify.3
+    spec:
+      nodeSelector:
+        master: "true"
+      containers:
+      - name: kubelet
+        image: quay.io/peanutbutter/hyperkube:v1.2.4_inotify.3
+        command:
+        - /nsenter
+        - --target=1
+        - --mount
+        - --wd=.
+        - --
+        - ./hyperkube
+        - kubelet
+        - --api-servers={{ index .APIServers 0 }}
+        - --config=/etc/kubernetes/manifests
```

Comment:
No, the difference is that this kubelet has the `master=true` label, so that it will also get "master" components if it is still running when its apiserver goes down.